WikidataOrg.php

```diff
@@ -71,3 +71,22 @@
 	$wgResourceModules = array_merge( $wgResourceModules, include( __DIR__ . '/resources/Resources.php' ) );
 
 };
+
+/**
+* Add a "Data access" link to the footer
+*
+* @param $skin SkinTemplate (from includes/SkinTemplate.php)
+* @param $template QuickTemplate (from includes/SkinTemplate.php)
+* @return bool
+*/
+$GLOBALS['wgHooks']['SkinTemplateOutputPageBeforeExec'][] = function( &$skin, &$template ) {
```

Comment:
This function should be moved to `WikidataOrg.hooks.php` (like the `BeforePageDisplay` hook above).